lib/WUI/http/httpd_structs.h

```diff
@@ -31,8 +31,9 @@ static const char *const g_psHTTPHeaderStrings[] = {
     "HTTP/1.0 500 Internal Server Error\r\n",
     "HTTP/1.0 401 Unauthorized\r\nWWW-Authenticate: ApiKey realm=\"401\"",
     "HTTP/1.0 304 Not Modified\r\n"
-    "HTTP/1.0 409 Conflict \r\n"
+    "HTTP/1.0 409 Conflict\r\n"
     "HTTP/1.0 415 Unsupported Media Type\r\n"
+    "HTTP/1.0 503 Service Temporarily Unavailable\r\n"
```

Comment:
The strings are missing commas between them, so they'll form one long string instead and the indexing will be out of bounds.